pkgs/applications/office/onlyoffice-bin/default.nix

```diff
@@ -0,0 +1,163 @@
+{ stdenv
+, lib
+, fetchurl
+
+, alsaLib
+, at-spi2-atk
+, atk
+, autoPatchelfHook
+, cairo
+, curl
+, dbus
+, dconf
+, dpkg
+, fontconfig
+, gdk-pixbuf
+, glib
+, glibc
+, gnome3
+, gst_all_1
+, gtk2
+, gtk3
+, libpulseaudio
+, libudev0-shim
+, makeWrapper
+, nspr
+, nss
+, pulseaudio
+, qt5
+, wrapGAppsHook
+, xkeyboard_config
+, xorg
+}:
+let
+
+  # Note on fonts:
+  #
+  # OnlyOffice does not distribute unfree fonts, but makes it easy to pick up
+  # any fonts you install. See:
+  #
+  # * https://helpcenter.onlyoffice.com/en/installation/docs-community-install-fonts-linux.aspx
+  # * https://www.onlyoffice.com/blog/2020/04/how-to-add-new-fonts-to-onlyoffice-desktop-editors/
+  #
+  # As recommended there, you should download
+  #
+  #     arial.ttf, calibri.ttf, cour.ttf, symbol.ttf, times.ttf, wingding.ttf
+  #
+  # into `~/.local/share/fonts/`, otherwise the default template fonts, and
+  # things like bullet points, will not look as expected.
+
+  # TODO: Find out which of these fonts we'd be allowed to distribute along
+  #       with this package, or how to make this easier for users otherwise.
+
+  noto-fonts-cjk = fetchurl {
```

Comment:
Please use the noto-fonts-cjk that are already in nixpkgs. https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/noto-fonts/default.nix#L77

Reply:
That seems to be something slightly different, and 10x much larger:

Nixpkgs' `noto-fonts-cjk`:

```
115M	/nix/store/rylkxlfxqp95z0hxyx6mqzfxvnfpw3ld-noto-fonts-cjk-2.001/share/fonts/opentype/noto-cjk/NotoSansCJK.ttc
```

This one used:

```
16M	/nix/store/zwpnc457ln4nf4brri9m2pcqa678cp48-onlyoffice-desktopeditors-6.1.0/share/desktopeditors/fonts/n1nd1bgihg5gwv2if7kcfqkbrlg8zwv4-NotoSansCJKsc-Regular.otf
```

I suppose `.ttc` is some kind of collection of multiple fonts?

Reply:
@GTrunSec You add fonts here: https://github.com/GTrunSec/onlyoffice-desktopeditors-flake/blob/6a404e6916a39e8b51211d64c438a3b25d139013/flake.nix#L70-L71

@LEXUGE You add fonts here:

https://github.com/icebox-nix/iceberg/blob/a49fa345b53cdf51a480cb53623a8194fb2d4b56/pkgs/office/onlyoffice-bin/default.nix#L67

Can any of you tell whether these fonts are strictly necessary in onlyoffice?

As far as I understand, OnlyOffice should pick up the system fonts and the ones in `~/.local/share/fonts`.

Do you think it would be OK to remove the fonts installation as done here so far, or did you find them to be necessary? Both of them seem to be for non-Latin alphabets, and I'm not really an expert at testing for those.

Reply:
The biggest problem with the OnlyOffice is that it does NOT correctly pick up the fonts 
present in the system. For other fonts, primarily CJK fonts, we have to include it. In addition to that, those added fonts are strict on format, and we tried in stock cjk fonts which don't work.

Reply:
@LEXUGE To clarify with "we tried in stock cjk fonts", you mean the `noto-fonts-cjk` package in nixpkgs with its `NotoSansCJK.ttc` file I mentioned above?

Reply:
Yes, the tic file in nixpkgs is what we tried.

Reply:
@LEXUGE Thanks, I've added a comment to the derivation about this.